frontend/public/components/routes.tsx

```diff
@@ -353,69 +324,86 @@ const RouteTargetRow: React.FC<RouteTargetRowProps> = ({ route, target }) => (
-            {ingress.wildcardPolicy}
-          </DetailsItem>
-          <DetailsItem
-            label="Router Canonical Hostname"
-            obj={route}
-            path="status.ingress.routerCanonicalHostname"
-          >
-            <div>{ingress.routerCanonicalHostname || '-'}</div>
-            {showCustomRouteHelp(ingress, route.metadata.annotations) && (
-              <CustomRouteHelp
-                host={ingress.host}
-                routerCanonicalHostname={ingress.routerCanonicalHostname}
-              />
-            )}
-          </DetailsItem>
-        </dl>
-        <h3 className="co-section-heading-secondary">Conditions</h3>
-        <Conditions conditions={ingress.conditions} />
-      </div>
-    ))}
-  </>
-);
+const RouteIngressStatus: React.FC<RouteIngressStatusProps> = ({ route }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      {_.map(route.status.ingress, (ingress: RouteIngress) => (
+        <div key={ingress.routerName} className="co-m-route-ingress-status">
+          <SectionHeading
+            text={`${t('network-route~Router: {{routerName}}', {
+              routerName: ingress.routerName,
+            })}`}
+          />
+          <dl>
+            <DetailsItem label={t('network-route~Host')} obj={route} path="status.ingress.host">
+              {ingress.host}
+            </DetailsItem>
+            <DetailsItem
+              label={t('network-route~Wildcard policy')}
+              obj={route}
+              path="status.ingress.wildcardPolicy"
+            >
+              {ingress.wildcardPolicy}
+            </DetailsItem>
+            <DetailsItem
+              label={t('network-route~Router canonical hostname')}
+              obj={route}
+              path="status.ingress.routerCanonicalHostname"
+            >
+              <div>{ingress.routerCanonicalHostname || '-'}</div>
+              {showCustomRouteHelp(ingress, route.metadata.annotations) && (
+                <CustomRouteHelp
+                  host={ingress.host}
+                  routerCanonicalHostname={ingress.routerCanonicalHostname}
+                />
+              )}
+            </DetailsItem>
+          </dl>
+          <h3 className="co-section-heading-secondary">{t('network-route~Conditions')}</h3>
+          <Conditions conditions={ingress.conditions} />
+        </div>
+      ))}
+    </>
+  );
+};
 
 const RouteDetails: React.FC<RoutesDetailsProps> = ({ obj: route }) => {
+  const { t } = useTranslation();
   const primaryIngressStatus: IngressStatusProps = getIngressStatusForHost(
     route.spec.host,
     route.status.ingress,
   );
   return (
     <>
       <div className="co-m-pane__body">
-        <SectionHeading text="Route Details" />
-        <RouteCharts namespace={route.metadata.namespace} route={route.metadata.name} />
+        <SectionHeading text={t('network-route~Route details')} />
+        <RouteCharts namespace={route.metadata.namespace} route={route.metadata.name} t={t} />
```

Comment:
We shouldn't be passing `t` to components. They should use their own `useTranslation`.

We'll eventually be passing a namespace when we call `useTranslation` so not all `t` are the same.

Reply:
ok, fixed